api/src/main/java/com/datatorrent/api/StreamingApplication.java

```diff
@@ -47,6 +47,12 @@
   String ENVIRONMENT = DT_PREFIX + "environment";
 
   /**
+   * Use this config variable to to add any external library to classpath for the DAG.
+   * Value of this variable is same as StramAppLauncher.LIBJARS_CONF_KEY_NAME.
+   */
+  String LIBJARS_CONF_KEY_NAME = "tmplibjars";
```

Comment:
Just looking at the string literal suggests something isn't right here. We configure applications through attributes, not random properties in the config file. Can you please check if there is a better way to express this dependency.